docs/apis/mos/pushapi_config_api.yaml

```diff
@@ -601,6 +601,315 @@ paths:
           $ref: '#/components/responses/VersionDoesntExist'
         410:
           $ref: '#/components/responses/VersionNoLongerValid'
+
+  /tokens/app/{app_id}:
+    put:
+      tags:
+        - Authentication Token
+      summary: Set Push API authentication token
+      parameters:
+        - name: app_id
+          in: path
+          description: Application ID
+          example: "id121244"
+          required: true
+          schema:
+            type: string
+      requestBody:
+        description: Body for validate URL
+        content:
+          application/json:
+            schema:
+              required:
+                - token
+              type: object
+              properties:
+                token:
+                  type: string
+                  maxLength: 600
+                  pattern: '^[ 	\-_.a-zA-Z0-9]*$'
+                  description: "The token value should be a string with a maximum of 600 characters, 
+                               containing only the characters a-z, A-Z, 0-9, and the special characters [ \t-_.]."
+                token_name:
+                  type: string
+                  nullable: true
+                  maxLength: 30
+                  pattern: '^X-[a-zA-Z0-9\-_]*$|^Authorization$|^$'
```

Comment:
This the correct was to do that in OAS 3:


          schema:
            oneOf:
              - type: string
                enum:
                  - liaz
              - type: string
                pattern: '^\w{10}$'
            minLength: 10